Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/CollectionEnums/SurfaceTypeEnum.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Collections
+{
+    /// <summary>
+    /// The type of surface to map the collect to.
+    /// </summary>
+    public enum SurfaceTypeEnum
```

Comment:
Some of these enums should probably be defined in the mrtk core

Reply:
Any guidance on organizational structure? _Core.Utilities.Enums? SortType and OrientType seem to be the most useful.

Reply:
`_Core.Definitions.Utilities`

Reply:
That's where most of our enums are defined.

Reply:
Ah okay, the naming convention changed. Got it.